src/util/pointer.h

```diff
@@ -36,10 +36,12 @@ class Buffer
 	Buffer(unsigned int size)
 	{
 		m_size = size;
-		if(size != 0)
+		if(size != 0) {
```

Comment:
very minor, in similar vein, could you format this as

```suggestion
		if (size != 0) {
```

same below.